src/utils/__init__.py

```diff
@@ -26,7 +26,7 @@ def parse_args():
     return params
 
 
-parse_args()
+# parse_args()
```

Comment:
Why did you do this? This call is needed for `GLOBAL_TENSORBOARD_WRITER` initialization. Revert this plz

```suggestion
parse_args()
```